tests/defaults.yml

```diff
@@ -7,12 +7,12 @@ parameters:
     defaultSourceNamespace: openshift-marketplace
     defaultSource: redhat-operators
   openshift4_keepalived:
-    namespace: openshift-operator-community
+    namespace: openshift-keepalived-operator
```

Comment:
Or remove it to test that the default value works as intended (since the test instance is caleld `defaults`)

Reply:
yup, removed the channel as well